diff --git a/dist/module.mjs b/dist/module.mjs
index 9b1f34bf9b525d8f0dbf450dfc7963d8d42ec43b..be9062d9b1fc9eaed8094d533790fe000d7214f4 100644
--- a/dist/module.mjs
+++ b/dist/module.mjs
@@ -13,6 +13,7 @@ const ipxSetup = async (_providerOptions, moduleOptions) => {
   }
   const ipxOptions = {
     dir: resolve(nuxt.options.srcDir, moduleOptions.dir || nuxt.options.dir.public),
+    maxAge: 2592000,
     domains: moduleOptions.domains,
     sharp: moduleOptions.sharp,
     alias: moduleOptions.alias